frontend/src/screens/PathPage/containers/PathPage/sagas.ts

```diff
@@ -1,11 +1,19 @@
 import { all, call, put, takeEvery } from 'redux-saga/effects';
+import  * as pathService from '@screens/PathPage/services/path.service';
 import { fetchPathDataRoutine } from '@screens/PathPage/routines';
 import { AnyAction } from 'redux';
+import { history } from '@helpers/history.helper';
 
 function* getPathData({ payload }: AnyAction) {
+  console.log('here');
   try {
-    yield put(fetchPathDataRoutine.success('SUCCESS'));
+    console.log(payload);
+    const response = yield call(() =>pathService.getPathData(payload));
+    console.log('resp');
+    console.log(response);
+    yield put(fetchPathDataRoutine.success(response));
   } catch (error) {
+    // history.push('/');
```

Comment:
Remove console.log, comments and replace `() =>pathService.getPathData(payload)` with `pathService.getPathData, payload`